tests/unit/test_install.py

```diff
@@ -228,7 +228,7 @@ def test_create_database(ws, caplog, mock_installation, any_prompt):
             raise e.errs[0]
 
     assert "Kindly uninstall and reinstall UCX" in str(failure.value)
-    wheels.upload_to_wsfs.assert_not_called()
+    wheels.upload_to_wsfs.assert_called_once()
```

Comment:
`wheels.upload_to_wsfs` is now called as job creation calls this method and is reached when deploying jobs in parallel to the database creation